jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -331,6 +332,29 @@ public void initializeDefaults(Object object)
     {
     }
 
+    /**
+     * Utility method to resolve a provided path against a base directory.
+     *
+     * @param baseDir the base directory
+     * @param destPath the destination path (can be relative or absolute, syntax depends on OS + FileSystem in use)
+     * @return String to resolved and normalized path, or null if baseDir or destPath is empty.
+     */
+    public static String resolvePath(String baseDir, String destPath)
+    {
+        if (StringUtil.isEmpty(baseDir) || StringUtil.isEmpty(destPath))
+            return null;
+
+        Path base = Paths.get(baseDir);
+
+        if (!Files.exists(base))
+        {
+            // Add warning to debug output
+            LOG.debug("Configured baseDir={} does not exist", baseDir);
```

Comment:
Wrap with `if (LOG.isDebugEnabled())` for consistency.

Reply:
Removed that logging entirely from the resolvePath method.
I was only using it during development time, at one point that method was even failing if the base dir didn't exist.